### PR TITLE
Work: new code

### PR DESCRIPTION
**Pull request by RachelAnnB**
-svgs now work
-header works responsively
-started working on the products section
-edited the gitignore

### CODE REVIEW

File: src/scss/styles.css [hunk 1]
```diff
@@ -5,81 +5,252 @@ All Styles
 body {
```
Comment (zastrow) [inline src/scss/styles.css]: I’m surprised to see a CSS file in the SCSS folder. Did you add this, or did the build system?

File: src/partials/_header.hbs [hunk 2]
```diff
@@ -1,16 +1,13 @@
...
-      <li class="list__item"><a href="#" class="item__link">Products</a></li>
-      <li class="list__item"><a href="#" class="item__link">FAQ</a></li>
-    </ul>
-
-    <ul class="nav__list right">
-      <li class="list__item"><a href="#" class="item__link">Retailers</a></li>
-      <li class="list__item"><a href="#" class="item__link">About</a></li>
-    </ul>
-  </nav>
+	<h1 class="site-title">
```
Comment (zastrow) [inline src/partials/_header.hbs:2]: Be sure to follow the Sparkbox coding practice of two space indentation. https://github.com/sparkbox/standard/tree/master/style/all_code

File: src/scss/styles.css (hunk 1 above)
Comment (RachelAnnB) [inline src/scss/styles.css]: Build system did that.

File: src/partials/_products.hbs [hunk 3]
```diff
@@ -2,43 +2,54 @@
 	<h2>Products</h2>
 
 	<!----------Toy Products---------->
```
Comment (zastrow) [inline src/partials/_products.hbs]: Cool little thing. If this is a comment for coders reading the source HBS file, you can use a handlebars comment instead `{{!-- Comments --}}`. Those will be visible in the HBS file, but not in the HTML file. Not a must, just an FYI thing.

File: src/partials/_header.hbs (hunk 2 above)
Comment (RachelAnnB) [inline src/partials/_header.hbs:2]: Ok, not sure why but I think GitHub is adding all that extra space. In Brackets it looks just fine.

File: src/scss/_header.scss [hunk 4]
```diff
@@ -2,69 +2,92 @@
...
-	padding: 1.75em 0 1.25em;
-	@media(min-width: 980px) {
-		display: none;
+	height: 2em; //32px
+	width: 1.75em; //28px
+	margin: 0 auto;
+	overflow: hidden;
+	text-indent: -9999em;
+
+	@media(min-width: $bp-medium) {
```
Comment (zastrow) [inline src/scss/_header.scss]: With regard to your break point variable names, see Ethan Muller’s article on this subject. Generally size (and device) named variables breakdown fairly quickly. https://seesparkbox.com/foundry/using_names_to_manage_dozens_of_breakpoints

File: src/scss/styles.css (hunk 1 above)
Comment (zastrow) [inline src/scss/styles.css]: oh no!

File: src/partials/_header.hbs (hunk 2 above)
Comment (zastrow) [inline src/partials/_header.hbs:2]: Might need to take a look at the settings in your editor. GitHub is _usually_ a reflection of what is pushed up.

File: src/scss/_header.scss (hunk 4 above)
Comment (RachelAnnB) [inline src/scss/_header.scss]: I removed the break point variables from my scss, so I'm no longer naming any of my media queries.

**Comment by zastrow** [comment]
Merged via CLI. For got to close via CLI.

**Pull request closed without merging**